Zend/tests/033.phpt

```diff
@@ -18,14 +18,44 @@ $arr[][]->bar = 2;
 ?>
 --EXPECTF--
 
-Notice: Undefined variable: arr in %s on line %d
+Notice: Undefined variable: arr in %s on line 3
```

Comment:
Putting line numbers in tests is usually not a very good idea, as adding comments, etc. will break the test. It should be done only if line number is important.